scripts/scil_bundle_label_map.py

```diff
@@ -86,224 +149,205 @@ def _build_arg_parser():
 def main():
```

Comment:
It would be best to split this function into multiple and send them into the proper modules.